package.json

```diff
@@ -3,41 +3,33 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "start": "npm-run-all -p start:*",
-    "start:ts": "cross-env REACT_APP_MOCK=true craco start",
-    "start:tekster": "node ./scripts/watch-tekster.js",
-    "build": "npm-run-all -s build:tekster build:ts",
-    "build:micro": "cross-env REACT_APP_MICRO=true npm-run-all -s build:tekster build:ts",
-    "build:ts": "craco build",
-    "build:tekster": "react-intl-bundler --no-lookup src/tekster src/tekster",
+    "start": "cross-env REACT_APP_MOCK=true craco start",
+    "build:micro": "cross-env REACT_APP_MICRO=true npm run build",
+    "build": "craco build",
     "lint": "tslint ./src/**/*.ts*",
     "lint:fix": "tslint ./src/**/*.ts* --fix",
-    "test": "npm run lint && npm run build:tekster && craco test --env=jsdom",
```

Comment:
Bare for sikkerhets skyld: trenger man ikke å bygge tekstene for å kjøre tester?

Reply:
Nei, det gjør man jo ikke når alle tekster ligger i en fil siden react-intl er fjernet... 🤦‍♂

Reply:
Stemmer! Blir det ikke deilig å slippe dette unødvendige steget? 🤩